MATLAB/pipeline/+tracking/JawTracking3DSid.m

```diff
@@ -11,11 +11,19 @@
     properties
         keySource = (experiment.getSchema().v.Session & 'rig = "RRig-MTL"')...
             & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 3"')...
-            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"');
+            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"')...
```

Comment:
this line was removed and now added in, is this intentional, if so, perhaps you mean

```
& (tracking.getSchema().v.Tracking & 'tracking_device in ("Camera 3", "Camera 4")')...
```

tracking_device being either `Camera 3` or `Camera 4` ?

Reply:
No, this was not intentional, I meant it must have tracking in Camera 3 AND Camera 4